src/ui/menu.cpp

```diff
@@ -630,6 +645,15 @@ bool MenuBox::onProcessMessage(Message* msg)
 
     case kKeyDownMessage:
       if (menu) {
+        MenuItem* child_with_submenu_opened = nullptr;
+        findChildWithSubmenuOpened(menu, child_with_submenu_opened);
+        if (child_with_submenu_opened &&
+            child_with_submenu_opened->inBar() &&
+            this->type() == kMenuBarWidget &&
+            msg->modifiers() == kKeyAltModifier) {
+          cancelMenuLoop();
+        }
+
```

Comment:
I've been testing the fix and it works great. The only thing I'm not sure about is this piece of code, which I couldn't figure out how to make this if condition true. I also tested the fix without this code and it seems to work as well. So I would love to know what I am missing here.

Reply:
You're right! Thank you!